scala/spark-wordcount/pom.xml

```diff
@@ -9,11 +9,12 @@
     <version>0.1</version>
 
     <properties>
-        <bigtable.version>1.7.0</bigtable.version>
-        <hbase.version>1.3.1</hbase.version>
+        <bigtable.version>1.15.0</bigtable.version>
+        <hbase.version>1.6.0</hbase.version>
+        <hadoop.version>2.8.5</hadoop.version>
```

Comment:
I think we should target the version of hadoop used by the current stable dataproc release (2.10.0)

Reply:
Ok, let me check that out...

Reply:
fixed